great_expectations/data_context/data_context/serializable_data_context.py

```diff
@@ -118,7 +118,7 @@ def _create(
         """
         Build a new great_expectations directory and DataContext object in the provided project_root_dir.
```

Comment:
should `great_expectations` in this line be similarly changed to `gx`?

Reply:
it's too far out of the diff for me to comment on but I have the same question about line 130 below which has

> project_root_dir: path to the root directory in which to create a new great_expectations directory

should `great_expectations` in that line be similarly changed to `gx`?

Reply:
Thanks @klavavej ! I'll try to take another look (this time a holistic look at the docstring) later today.